<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userappeal</class>
 <widget class="QWidget" name="userappeal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>671</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previos">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>201</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>browse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
